api/samples/stl_test.cpp

```diff
@@ -71,6 +71,7 @@ dr_init(client_id_t client_id)
 {
     int i;
     bool success = true;
+    (void)success;
```

Comment:
This seems to have no relation to the rest of the PR: a mistake?  Supposed to be in another PR I assume.

Reply:
Also temporary change, will get removed (debugging clang error).

Reply:
I would suggest following the habit of always marking temp changes as such with comments, esp in shared branches.  Multiple benefits: easier to remember what needs to be reverted once done debugging; easier for someone else to pick up the PR if necessary.

Reply:
Ack/Done.

Reply:
This is what `NOCHECKIN` was meant for, and we have checks to ensure a line which contains `NOCHECKIN` (usually at the end `// NOCHECKIN`) does not get merged.  For testing on Travis though it will fail that check up front.

Reply:
Quoting from https://github.com/DynamoRIO/dynamorio/wiki/Code-Style:

```
Mark any temporary or unfinished code unsuitable for committing with a NOCHECKIN comment. The make/codereview.cmake script will remind you to clean up the code.

 x = 4; /* NOCHECKIN: temporary debugging change */
```

Reply:
Done.